src/MediaWiki/Search/Search.php

```diff
@@ -4,6 +4,7 @@
 
 use Content;
 use DatabaseBase;
+use MediaWiki\MediaWikiServices;
```

Comment:
Access, reference, and usage to `MediaWiki\MediaWikiServices` should be confined to either `SMW\Tests\TestEnvrionment` [0] or `ApplicationFactory`.

[0] https://github.com/SemanticMediaWiki/SemanticMediaWiki/blob/master/tests/phpunit/TestEnvironment.php#L95-L114